### ISSUE TITLE
Increased fetal lens echogenicity

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7444]
**Preferred term label:**
Increased fetal lens echogenicity

**Synonyms**

**Definition (free text, please give PubMed ID)**
Def: Increased echogenicity (increased brightness in the gray-scale sonography image) of the lens of a fetus.
Comment: Both the normal fetal lens and vitreous appear hypoechoic by ultrasound. On endovaginal ultrasound, the fetal lens may be visualized as early as 14 weeks’ gestational age. The outline of the lens may be visible as a thin, echogenic line in the anterior aspect of the globe; however, the lens may be difficult to see depending on the angle of the insonation beam relative to the lens position. Increased echogenicity of the fetal lens can be observed with some forms of developmental cataract.

PMID:27433255

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
